content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus was [sunset](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The sunset of OpenCensus will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
+plans to use OpenTelemetry exporters with the OpenTelemetry collector to provide matching telemetry capabilities.
+
+This is an advance notice of Istio deprecating OpenCensus, tentatively scheduled for Oct. 10, 2024.
```

Comment:
This date has sailed past.  Is this still relevant, @lei-tang ?

Reply:
As Istio 1.20 release candidate is out on 11-2-2023 https://gcsweb.istio.io/gcs/istio-release/releases/1.20.0-rc.0/, I think the work to remove opencensus may take more time to complete (such as Istio 1.21 or 1.22). Do we have any volunteers to work on removing opencensus?